lib/alephant/lookup.rb

```diff
@@ -9,9 +9,9 @@ module Lookup
     include Logger
     @@lookup_tables = {}
 
-    def self.create(table_name)
+    def self.create(table_name, config={})
```

Comment:
I am thinking of passing in Broker config here. I could pass in settings needed for the cache, but that might become quite a long constructor. I have the options of:
1) Keep as above.
2) Pass in table_name in config too, as it's in there as well.
3) Pass in params separately.
4) Somehow read the Broker config in this gem. Not sure if that is possible.